WcaOnRails/app/views/competitions/_nav.html.erb

```diff
@@ -169,7 +169,14 @@
 
     </div>
     <div id="competition-data">
-      <h3><%= @competition.name %></h3>
+      <h3>
+        <% if @competition.championships.any? %>
+          <span class="championship-trophy" data-toggle="tooltip" data-placement="bottom" title="<%= @competition.championships.map { |championship| championship.name }.join(", ") %>">
```

Comment:
`Championship#name` is not internationalized yet, if we want to display that on competition pages, we definitely want to do so now.

Reply:
Good call. I'm having trouble understanding how the country names are internationalized, since they aren't in the translation yamls. Do I need to query the database to get those?

Reply:
We have this gem `i18n-country-translations` which I believe handles most names, and some additional keys in the `en.yml` file: https://github.com/thewca/worldcubeassociation.org/blob/793e9cd4a2e391a41593b7bb5761e43395049500/WcaOnRails/config/locales/en.yml#L24-L31
The `Country` model includes `LocalizedSortable` which in turn gives it this method: https://github.com/thewca/worldcubeassociation.org/blob/793e9cd4a2e391a41593b7bb5761e43395049500/WcaOnRails/app/models/concerns/localized_sortable.rb#L10-L12
So basically `Country#name` is internationalized, but there's also `Country#name_in` if you want to force some language (e.g. just English for non-i18n parts).

Reply:
That's very helpful, thank you. So it looks like continent.name actually already calls the method you're talking about, but I think unfortunately those translations aren't always going to work since they're in the nominative case. Do we already have some way of dealing with this situation? For example, in Russian you might say "Чемпионат Индии" (genitive case), while the translation for "India" is "Индия", which is in the nominative case.

Reply:
Yeah, "national championship: [country]" or "National championship of [country]" sounds good to me!

Reply:
"National Championship for [country]" works great in English and some other languages, but it doesn't work in languages with a case system (such as Russian or German). In languages like this, the noun is modified (in part) according to the preposition, so "[country]" and "of [country]" require different translations of the country. This is a rough analogy, but it would sound something like "National Championship for French" for example.

"National championship: [country]" is a little awkward, but could work. The alternative is adding a bunch of translation keys for the countries in the genitive case, but that is quite a bit of work for translators.

Reply:
Ah, ok! How about we do "National championship: [country]" for now, and we can decide later if we want to make our translators do a bunch of these translations to make it less awkward sounding.